client/src/components/DatasetInformation/DatasetError.test.js

```diff
@@ -61,4 +61,24 @@ describe("DatasetError", () => {
         expect(wrapper.findAll("#dataset-error-has-duplicate-inputs").length).toBe(0);
         expect(wrapper.findAll("#dataset-error-email").length).toBe(0);
     });
+
+    it("success message hides form fields and button", async () => {
```

Comment:
The idea with these descriptions is that you can complete the it ... into a sentence,
so `it("hides form fields and button on success")`

Reply:
Yeah, I know- I'm used to/prefer `test` instead, so I forgot to adjust the wording for `it`.